backend/src/main/scala/cromwell/backend/package.scala

```diff
@@ -0,0 +1,35 @@
+package cromwell
+
+import com.typesafe.config.Config
+import cromwell.core.{WorkflowId, WorkflowOptions}
+import wdl4s._
+import wdl4s.values.WdlValue
+
+package object backend {
+
+  /**
+    * For uniquely identifying a job which has been or will be sent to the backend.
+    */
+  case class BackendJobDescriptorKey (callFqn: String, index: Option[Int], attempt: Int)
+
+  type BackendSymbolMap = Map[FullyQualifiedName, WdlValue]
+
+  /**
+    * For passing to a BackendWorkflowActor for job execution or recovery
+    */
+  //case class BackendJobDescriptor(key: BackendJobDescriptorKey, call: Call, locallyQualifiedInputs: CallInputs)
+  case class BackendJobDescriptor(key: BackendJobDescriptorKey, call: Call, symbolMap: BackendSymbolMap)
+
+  /**
+    * For passing to a BackendActor construction time
+    */
+  case class BackendWorkflowDescriptor(id: WorkflowId,
+                                       workflowOptions: WorkflowOptions,
+                                       workflowNamespace: NamespaceWithWorkflow,
+                                       declarations: Map[String, WdlExpression])
```

Comment:
Is there any difference between this `declarations` and the `workflowNamespace.workflow.declarations`?